js/manual-version-switch.js

```diff
@@ -0,0 +1,26 @@
+$(window).load(function() {
+  let project = window.location.pathname.split('/')[2];
+  let channel = window.location.pathname.split('/')[3];
+  let unstable = $("<a/>")
+    .attr("href", "/manual/" + project + "/unstable")
+    .text("unstable");
+  let stable = $("<a/>")
+    .attr("href", "/manual/" + project + "/stable")
+    .text("stable");
+  let channels = $("<dl>")
+    .append($("<dt/>").text("Versions"))
+    .append($("<dd/>").append(unstable))
+    .append($("<dd/>").append(stable));
+  let current = $("<div/>")
+    .click(function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+        $(this).parent().toggleClass("opened");
+    })
+    .text("v: " + channel);
```

Comment:
```
<i class="fa fa-caret-down" aria-hidden="true"></i>
```

Ideally with the proper rules to switch up "up" as needed

```
<i class="fa fa-caret-up" aria-hidden="true"></i>
```

Additionally, I don't know where the caret should be, right or left? A "proper" dropdown places it at the right, but this is not a "proper" dropdown!

I would also argue that + and - would be proper (too) since this is an expandable box

```
<i class="fa fa-plus" aria-hidden="true"></i>
<i class="fa fa-minus" aria-hidden="true"></i>
```

They also have a square variant if it makes sense here.

Tip: use the fontawesome 4 page to search for icons:

 * https://fontawesome.com/v4.7.0/icons/

Reply:
I'll use + and - as you suggested then we can place it anywhere we want and see fit best.